src/lints/function_const_removed.ron

```diff
@@ -48,4 +48,9 @@ SemverQuery(
     },
     error_message: "A publicly-visible function is no longer `const` and can no longer be used in a `const` context.",
     per_result_error_template: Some("function {{join \"::\" path}} in file {{span_filename}}:{{span_begin_line}}"),
+    witness: (
+        hint_template: r#"const fn foo() {
```

Comment:
We're still setting up norms for how we do witnesses, but let's try to name witness functions and types things like `witness` rather than `foo` whenever possible? Names like `foo` can be mildly problematic in some circumstances, and in this case we have a better name available so let's use it.
```suggestion
        hint_template: r#"const fn witness() {
```